samples/test/support/management-api/a18nClient.ts

```diff
@@ -38,6 +39,22 @@ class A18nClient {
     return match?.groups?.code;
   }
 
+  async getSmsCode(profileId: string) {
+    let retryAttemptsRemaining = 5;
+    let response;
+    while (!response?.content && retryAttemptsRemaining > 0) {
+      await waitForOneSecond();
+      response = await this.getOnURL(LATEST_SMS_URL.replace(':profileId', profileId)) as Record<string, string>;
+      --retryAttemptsRemaining;
+    }
+
+    const match = response?.content?.match(/Your verification code is (?<code>\d+)/);
+    if (!match) {
```

Comment:
if (!match?.groups?.code) {
  throw err;
}

looks like this function should only return `code`, otherwise it throws.